### PR TITLE
170 Truncate tail of long name and email strings

### PR DESCRIPTION
**Pull request by zamorastp**
Screenshots are available in the second PR.

### CODE REVIEW

File: Sources/AccountSDKIOSWeb/Simplified Login/View/Sub Views/ContinueButton.swift [hunk 1]
```diff
@@ -33,16 +33,53 @@ final class ContinueButton: UIButton {
...
+        view.lineBreakMode = .byTruncatingTail
+        view.textColor = .white
+        view.translatesAutoresizingMaskIntoConstraints = false
+
+        return view
+    }()
+    
+    private lazy var combinedTitleView: UIView = {
+        return UIView()
+    }()
```
Comment (oscarsilver) [inline Sources/AccountSDKIOSWeb/Simplified Login/View/Sub Views/ContinueButton.swift:52]: Could we use UIStackView for this instead to be able to skip adding some constraints?

File: Sources/AccountSDKIOSWeb/Simplified Login/View/Sub Views/ContinueButton.swift [hunk 2]
```diff
@@ -33,16 +33,53 @@ final class ContinueButton: UIButton {
...
          initialsLabel.centerXAnchor.constraint(equalTo: avatarView.centerXAnchor),
-         avatarView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 12),
+         avatarView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 8),
          avatarView.centerYAnchor.constraint(equalTo: centerYAnchor),
          avatarView.widthAnchor.constraint(equalToConstant: 40),
          avatarView.heightAnchor.constraint(equalToConstant: 40),
-         titleLabel!.leadingAnchor.constraint(equalTo: avatarView.trailingAnchor, constant: 10),
-         titleLabel!.topAnchor.constraint(greaterThanOrEqualTo: topAnchor, constant: 3),
-         titleLabel!.bottomAnchor.constraint(lessThanOrEqualTo: bottomAnchor, constant: -3)
+         nameTitleLabel.leadingAnchor.constraint(equalTo: avatarView.trailingAnchor, constant: 8),
```
Comment (oscarsilver) [inline Sources/AccountSDKIOSWeb/Simplified Login/View/Sub Views/ContinueButton.swift:75]: Suggestion:

I usually add a small DSL extension to projects for handling autolayout constraints. (Kinda like in [this blog post](https://www.swiftbysundell.com/articles/building-dsls-in-swift/). 

Not for this PR but maybe something we can look into as a refactoring thing?

**Review by oscarsilver — APPROVED** [review]
LGTM

File: Sources/AccountSDKIOSWeb/Simplified Login/View/Sub Views/ContinueButton.swift (hunk 1 above)
Comment (zamorastp) [reply Sources/AccountSDKIOSWeb/Simplified Login/View/Sub Views/ContinueButton.swift]: Done

File: Sources/AccountSDKIOSWeb/Simplified Login/View/Sub Views/ContinueButton.swift (hunk 2 above)
Comment (zamorastp) [reply Sources/AccountSDKIOSWeb/Simplified Login/View/Sub Views/ContinueButton.swift]: Yes, we can think about it but maybe when we decide which version of SL is our final version? :)

**Pull request merged**